R/kernel.r

```diff
@@ -198,9 +198,11 @@ main <- function(connection_file="") {
 
 #'Install the kernelspec to tell IPython (>= 3) about IRkernel
 #'
+#'@param user Install into user directory ~/.ipython or globally?
 #'@export
-installspec <- function() {
+installspec <- function(user=F) {
```

Comment:
No problem. Actually, the default was originally to do a per-user install, until a change I made in IPython that switched the default. So this is actually bringing the default back to what I originally intended.